### ISSUE TITLE
Not able to set any value or send any command

### ISSUE THREAD

**Issue by gtorino** [twocolors/node-red-contrib-midea-hvac#4]
Hello, i'm able to **get** all device values when I inject a timestamp. But if I inject a json payload, like this:
{
    "powerState": true,
    "targetTemperature": 25,
    "operationalMode": 4,
    "fanSpeed": 80,
    "swingMode": 0,
    "ecoMode": false,
    "turboMode": false
}
I **always** got the error: "_Midea: 3176:The asyn reply does not exist_ "
What am I doing wrong?
On picture, "Accendi" stands for _turn on_, "Spegni" stands for _turn off_.
Many thanks in advance.
Giuseppe
![image](https://user-images.githubusercontent.com/18402703/113519995-4d524a80-9590-11eb-91eb-7c74bd2900d0.png)

**Comment by twocolors** [comment]
oh, can try send one diff elm in payload

**Comment by gtorino** [comment]
Thanks, I tried several combinations, even just powerState and targetTemperature only, with no success. I initially get the error "wrong command or device unreachable" and after a few tries, the 3176 error. Note that, to me, the swingMode field isn't a boolean but a number (0,1, etc). I tried both. When I send a empty payload I get: 
{loginId: "xxxxxxxxxxxxxxxxxxxxxx"
sessionId: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
accessToken: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
userId: "xxxxxxxxxx"
dataKey: "xxxxxxxxxxxxxxxxxxx"
powerState: false
imodeResume: false
timerMode: false
applianceError: false
targetTemperature: 22
operationalMode: 4
fanSpeed: 102
onTimer: object
offTimer: object
swingMode: 0
cozySleep: 0
save: false
lowFrequencyFan: false
superFan: false
feelOwn: false
childSleepMode: false
exchangeAir: false
dryClean: false
auxHeat: false
ecoMode: false
cleanUp: false
tempUnit: false
sleepFunction: false
turboMode: false
catchCold: false
nightLight: false
peakElec: false
naturalFan: false
indoorTemperature: 22.5
outdoorTemperature: 14.5
humidity: 10 }
